framework/rpc/grpc_channelz.py

```diff
@@ -130,7 +130,7 @@ def find_channels_for_target(
         return (
             channel
             for channel in self.list_channels(**kwargs)
-            if channel.data.target == target
+            if target in channel.data.target
```

Comment:
All good, just let's add a comment above the return saying we do `in` check because the `dns:///` schema fallback.

Reply:
done

Reply:
Looks great!